frontend/src/features/projects/spl/forms/SurplusPropertyListForm.scss

```diff
@@ -14,7 +14,7 @@
   .form-check-label {
     font-size: 12px;
     font-family: 'BCSans', Fallback, sans-serif;
-    padding-right: 1rem;
```

Comment:
In most cases margin should be used to provide gaps between adjacent fields not padding.